# Request
---
# Response
int16[6] statusvalue
bool success
